### DISCUSSION TITLE
ERROR: (5502) : Could not load the CVE OVAL for 'XENLAL'. 'XMLERR'：Bad attribute closing for 'encoding'='utf-8'.'

### DISCUSSION THREAD

**Discussion by 755914956** [wazuh/wazuh#14761] [Q&A] [answered]
How to solve the above error

**Comment by mdiego92** [comment, marked as answer]
Hello 755814956,

The issue might be related to the .XML file for the SCA or OpenSCAP configuration depending on your Wazuh version.
Could you please confirm what version of Wazuh are you using?

Thank you very much.

**Reply by mdiego92** [reply]
Hello 755814956,

I forgot to ask you, are you using an offline feed update configuration?
Also, it would be great if you could share with us the full ossec.log file.

Please let us know if you have any questions or doubts.

Thank you!

**Reply by 755914956** [reply]
Hello mdiego92
current use  latest version. I have a new problem.

I used offline update configuration.Here is my configuration and the error message

`ossec.conf:
<! - - Ubuntu OS vulnerabilities
‹provider name="canonical">
<enabled>no</enabled>
<os>bionic</05>
<os>focal</os>
<0s> jammy</05>
<update interval>1h</update interval>
</provider>
<!- - Debian OS vulnerabilities
‹provider name="debian">
<enabled>no</enabled>
<ossstretch</os>
<os>buster</os>
<os>bullseye</05>
<update interval>h</update interval>
</provider>

<!- - RedHat OS vulnerabilities
<provider name="redhat">
<enabled»yes</enabled>
‹os path="/var/ossec/rhel-7-including-unpatched.oval.xml.bz2">7</os>
<update_interval>lh</update_interval>
</provider>

<!- - Amazon Linux OS vulnerabilities
‹provider name="alas">
<enabled>no</enableds
<os>amazon-linux</os>
<os>amazon-linux-2</os>
<update_interval»lh</update_interval>`


`error
2022/09/01 21:50:26 wazuh-modulesd:vulnerability-detector:INFO:(5400): Starting "Red Hat Enterprise Linux 7' database update.
2022/09/01 21:50:38 wazuh-modulesd:vulnerability-detector: INFO: (5430): The update of the 'Red Hat Enterprise Linux 7' feed finished successfully.
2022/09/01 21:55:26 wazuh-modulesd:vulnerability-detector: INFO: (5400): Starting 'JSON Red Hat Enterprise Linux' database update.
2022/09/01 22:06:32 wazuh-modulesd:vulnerability-detector:WARNING: (5547) : There was no valid response to
•https://access.redhat.com/labs/securitvdataapi/cve.
ison?after=1999-01-01&per page=
1000&page=1°
after
3' attempts. Trying the next page.
2022/09/01 22:17:39 wazuh-modulesd:vulnerability-detector: WARNING: (5547): There was no valid response to 'https:L/access.redhat.com/labs/securitydataapi/cve.ison?after=1999-01-014per page=
1000&page=2°
after
attempts. Trying the next page.
2022/09/01 22:28:45 wazuh-modulesd:vulnerability-detector:WARNING:(5547): There was no valid response to 'httpsAL/access. redhat.com/labs/s
iter=1999-01-01&per page=
1000&page=3°
after
'3' attempts. Trying the next page.
2022/09/01 22:39:51 wazuh-modulesd:vulnerability-detector: WARNING: (5547) : There was no valid response to •https:/access
fter=1999-01-01&per page=
1000&page=5°
after
131
attempts. Trying the next page.
2022/09/01 23:02:04 wazuh-modulesd:vulnerability-detector:WARNING:(5547): There was no valid response to 'https:/Laccess.redhat.com/labs/securitydataapi/cve.json?after=1999-01-01&per page=
1000&page=6°
after
'3'
attempts. Trying the next page.
2022/09/01 23:02:04 wazuh-modulesd:vulnerability-detector:ERROR:(5553): The allowed number of failed pages (5) has been exhausted. The feed will not be updated.
2022/09/01 23:02:04 wazuh-modulesd:vulnerability-detector:ERROR:(5513):CVEdatabasecouldnotbeupdated.
2022/09/01 23:02:05 wazuh-modulesd:vulnerability-detector:INFO:
2022/09/01 23:02:17 wazuh-modulesd:vulnerability-detector:INFO:(5430): The update of the. 'Red Hat Enterprise Linux 7°
(5200) : starting "Red Hat Enterprise Linux 7. database #edat finished successfully.
2022 109107 33:02: 1% wazuh modulead vilnerability: detector: ENFor. 90302). Unavitlabi/norahaltitiftos at the Nvo databaso. The scan is aborted.
2022/09/01 23:02:17 wazuh-modu
nerabilitv-detector:
2022/09/01 23:07:17 wazuh-modulesd
actor.
2022/09/01 23:07:17 wazuh-modulesd
vulnerability-de
ENFOi. ($0902). Unatitladi/nerahil;tiLiffoat the Nvo database. The scan is aborted.
2022/09/01 23:12:17 wazuh-modulesd:vulnerability-detector
2022/09/01 23:12:17 wazuh-modulesd: vulnerabl
1w-detector.
Enfon. 99901. Staavitt.bi/nerahalitiuifto at the No database. The scan is aborted.
2022/09/01 23:17:17 wazuh-modulesd:vulnerabl
2022/09/01 23:17:17 wazuh-modulesd:vulnerabi
The
scan is aborted.
2022/09/01 23:22:17 wazuh-modulesd:vulnerability
2022/09/01 23:22:17 wazuh-modulesd: vulnerabi
ENFOi. '99002j. Staaväniabtilneraholat/iff"; at the NVD database,
The scan is aborted.
2022/09/01 23:27:17 wazuh-modulesd:vulnerab
The scan is aborted.
2022/09/01 23:27:17 wazuh-modulesd:
2022/09/01 23:32:17 wazuh-modulesd
2022/09/01 23:32:17 wazuh-modu
Enron. '99302). StantäRitat/herchälätxt-8%0; at the wvo database. The scan is aborted.
23:37:17 wazuh-modu
3932,/80/87 Zaria, varn nocules
octor:
INFO: (5431):
Starting vulnerability scan.
ector:
ERROR:
(5582) :
Unavailable vulnerabilities at the NVD database. The scan is aborted.
3832/89/79 33:827
wazun-modulesd-vulnerabi
etector
INFO:
(5431): Starting vulnerability
wazuh-modulesoevulr
oranslatv-detector:
ERROR:
(5582): Unavailable vulnerabilities at the NVD database. The scan is aborted.
`

**Reply by mdiego92** [reply]
Hello 755814956,

It seems that the information you shared with us is partially corrupted. If possible could you please attach both files to your message?

We would like to suggest adding the `<?xml version="1.0" encoding="UTF-8"?>` parameter in your cve-ubuntu-xenial-oval.xml file.
Another thing you could try is changing it to online feeds and try again.

Finally, we would recommend enabling the NVD feed, because it seems you don't have enabled it.
**Article for reference**: https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/offline-update.html#national-vulnerability-database

Please let us know if you have any questions or doubts.

Thank you very much.
